### PR TITLE
chore: Update security docker images page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Updated the copy on /security/docker-images

## QA
- Go to https://ubuntu-com-14565.demos.haus/security/docker-images
- Check that the changes requested in [the copy doc](https://docs.google.com/document/d/1zZibtjU141e4mBxo_0PtfYxJ6e5fyaaAdFZ6v945XAM/edit?tab=t.0#heading=h.b4oh6xkbn2jq) have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-17711

### CODE REVIEW

File: templates/security/docker-images.html [hunk 1]
```diff
@@ -87,14 +107,14 @@ <h2 class="u-sv3">FAQ on the LTS Docker Image Portfolio</h2>
...
     <p class="p-heading--4"><strong>Can I enable FIPS mode on Ubuntu-based container images?</strong></p>
-    <p>Yes, with a valid Ubuntu Advantage subscription. Hosts or nodes running the hardened Ubuntu-based container images must be covered with Ubuntu Advantage subscriptions or be entitled Ubuntu Pro machines. You can read more about how to enable FIPS mode on container images <a href="/blog/fips-ubuntu-container-security-updates">in this blog post</a>.</p>
+    <p>Yes, with a valid Ubuntu Pro subscription. Hosts or nodes running the hardened Ubuntu-based container images must be covered with Ubuntu Advantage subscriptions. You can read more about how to enable FIPS mode on container images <a href="/blog/fips-ubuntu-container-security-updates">in this blog post</a>.</p>
```
Comment (britneywwc) [inline templates/security/docker-images.html:117]:
```suggestion
    <p>Yes, with a valid Ubuntu Pro subscription. Hosts or nodes running the hardened Ubuntu-based container images must be covered with Ubuntu Pro subscriptions. You can read more about how to enable FIPS mode on container images <a href="/blog/fips-ubuntu-container-security-updates">in this blog post</a>.</p>
```

**Review by britneywwc — CHANGES_REQUESTED** [review]

File: templates/security/docker-images.html [hunk 2]
```diff
@@ -73,6 +73,26 @@ <h2>Our Commitment</h2>
...
+  <div class="row u-equal-height">
+    <div class="col-4 u-hide--medium u-hide--small u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/8b0a474a-Microsoft_.NET_logo.svg",
+        alt="Microsft .NET logo",
+        width="230",
+        height="230",
+        hi_def=True,
+        loading="auto"
+      ) | safe }}
```
Comment (britneywwc) [inline templates/security/docker-images.html:86]:
```suggestion
      {{ image(url="https://assets.ubuntu.com/v1/8b0a474a-Microsoft_.NET_logo.svg",
                      alt="Microsft .NET logo",
                      width="230",
                      height="230",
                      hi_def=True,
                      loading="auto") | safe }}
```

**Comment by steverydz** [comment]
Thanks for the review @britneywwc. I've committed your suggestions

**Review by britneywwc — APPROVED** [review]
Thanks, looks great!

**Pull request merged**